### PR TITLE
Update README.md for the pyembree

### PR DESCRIPTION
**Pull request by yw5aj**
On my computer it says:

```
C:\Users\shwang>conda install -c conda-forge pyembree
Fetching package metadata .............


PackageNotFoundError: Package missing in current win-64 channels:
  - pyembree

Close matches found; did you mean one of these?

    pyembree: embree



C:\Users\shwang>conda install -c conda-forge embree
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment C:\Users\shwang\AppData\Local\Continuum\Anaconda3:

The following NEW packages will be INSTALLED:

    embree:    2.14.0-0      conda-forge

The following packages will be SUPERSEDED by a higher-priority channel:

    conda:     4.3.16-py36_0             --> 4.2.13-py36_0 conda-forge
    conda-env: 2.6.0-0                   --> 2.6.0-0       conda-forge

Proceed ([y]/n)? y

conda-env-2.6. 100% |###############################| Time: 0:00:00  40.52 kB/s
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,4 +157,4 @@ If you want to deploy something in a container that uses trimesh, automated buil
 
 Or, for a much smaller image with no boolean operations and slightly slower graph operations (no graph-tool installed, trimesh will fall back to scipy or networkx):
 
-`docker pull mikedh/trimesh_minimal`
```
Comment (yw5aj) [inline README.md]: Oops. I didn't realize that I touched Line 160.

**Comment by mikedh** [comment]
Hey, good find thanks! I didn't notice that pyembree was osx/linux only: though I think the `embree` package is just the C/C++ package, without the `pyembree` bindings (if you try `import pyembree` in a python shell does it work?) .  I should probably remove it from the default install though, since it is not platform generic.

**Comment by yw5aj** [comment]
@mikedh Thank you for the catch! You are totally right - it only installs the C++ library of `embree`. When I do `import pyembree` it would not work. I should've checked that in the first place :P

I just changed the PR to remove it from the default install!

**Comment by mikedh** [comment]
LGTM, thanks!

**Comment by yw5aj** [comment]
Thank YOU for the great library! :)

**Pull request merged**